pages/index.tsx

```diff
@@ -1,11 +1,11 @@
 import Head from "next/head";
 import { Inter } from "next/font/google";
-import Navbar from "@/components/NavBar/Navbar";
-import About from "@/components/About";
-import Main from "/components/Main.jsx";
-import Skills from "@/components/Skills.jsx";
-import Projects from "@/components/Projects/Projects";
-import Contact from "@/components/Contact";
+import Navbar from "../components/NavBar/Navbar";
```

Comment:
revert back to `@`
https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json